worker/runtime/spec/namespaces.go

```diff
@@ -32,11 +35,7 @@ func OciNamespaces(privileged bool) []specs.LinuxNamespace {
 	return PrivilegedContainerNamespaces
 }
 
-func cgroupNamespacesEnabled() {
-	if _, err := os.Stat("/proc/self/ns/cgroup"); os.IsNotExist(err) {
-		return
-	}
-
-	UnprivilegedContainerNamespaces = append(UnprivilegedContainerNamespaces,
-		specs.LinuxNamespace{Type: specs.CgroupNamespace})
+func cgroupNamespacesSupported() bool {
+	_, err := os.Stat("/proc/self/ns/cgroup")
+	return !os.IsNotExist(err)
```

Comment:
Actually, I think this logic is a bit off - if an error occurs that is not `os.ErrNotExist`, it'll return true. I think it should be something like:

```go
_, err := os.Stat("/proc/self/ns/cgroup")
if err != nil {
	return !os.IsNotExist(err)
}
return true
```

Also, looking at the [godoc](https://golang.org/pkg/os/#IsNotExist):

> This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, os.ErrNotExist).

So, I guess the preferred form is `errors.Is(err, os.ErrNotExists)`